Linear Regression

Anjali001 opened the pull request:
#10 
I've added the required changes. Let me know, if there's any issue.

Anjali001 committed:
Add files via upload

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Delete Linearreg-1.png

Anjali001 committed:
Delete regress.jpg

Anjali001 committed:
Add files via upload

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Add files via upload

Anjali001 committed:
Updating Linear Regression

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Update linearreg.md

amansharma2910 reviewed (changes requested):
Great work on the content @Anjali001. Before I merge your PR, I need you to make a few changes, as listed below:
 - [x] Add breaks between paragraphs. Right now the content looks a bit congested.
 - [x] The image with the fomulae is not centered and has poor spacing. If possible, add some padding to the image.
 - [x] If possible, add some more content explanation about regression and supervised learning in general. Right now the explanation is concise and to the point, but we need to content for learners who are beginners.
 - [x] Mention the resources that you referred to, if any.

Anjali001 commented:
@amansharma2910 Sure. I'll do the required changes asap.

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Delete LR_Formula.bmp

Anjali001 committed:
Add files via upload

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Delete regress.jpg

Anjali001 committed:
Add files via upload

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Update linearreg.md

Anjali001 committed:
Update linearreg.md

Anjali001 commented:
@amansharma2910 I've done the changes you requested. Please check and let me know if there's an issue. Also, I've added some content from my own article on medium (I've added it as reference too), I hope that's not a problem.

amansharma2910 committed:
Make few modifications
